Cargo.toml

```diff
@@ -33,7 +33,7 @@ intervaltree    = "0.2.6"
 jmespatch       = { version = "^0.3", features = ["sync"], optional = true }
 libflate        = "^1.0"
 log             = "^0.4"
-openidconnect   = { version = "^2.0.0-alpha", optional = true, default_features = false }
+openidconnect   = { version = "^2.0.0-alpha.3", optional = true, default_features = false }
```

Comment:
This is not really anything to do with this PR, I just snuck it in here as I saw that the dependency can still automatically pick up the next alpha which could contain breaking changes and so should be pinned to the .3 version which is known to work with our code.